app/javascript/components/patient/common_exposure_cohorts/CommonExposureCohortModal.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Form, Modal } from 'react-bootstrap';
+
+const COHORT_TYPES = [
+  '',
+  'Adult Congregate Living Facility',
+  'Child Care Facility',
+  'Community Event or Mass Gathering',
+  'Correctional Facility',
+  'Group Home',
+  'Healthcare Facility',
+  'Place of Worship',
+  'School or University',
+  'Shelter',
+  'Substance Abuse Treatment Center',
+  'Workplace',
+  'Other',
+];
+
+class CommonExposureCohortModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      common_exposure_cohort: props.common_exposure_cohort || {},
+      showCohortModal: false,
+    };
+  }
+
+  handleChange = event => {
+    if (event?.target?.value?.length <= 200) {
+      event.persist();
+      this.setState(state => {
+        return { common_exposure_cohort: { ...state.common_exposure_cohort, [event?.target?.id]: event?.target?.value } };
+      });
+    }
+  };
+
+  render() {
+    return (
+      <Modal size="lg" backdrop="static" show onHide={this.props.onHide} centered>
+        <Modal.Header closeButton>
+          <Modal.Title>{this.props.common_exposure_cohort ? 'Update' : 'Add'} Common Exposure Cohort</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Form.Group controlId="cohort_type">
+            <Form.Label className="input-label">Cohort Type</Form.Label>
+            <Form.Control
```

Comment:
These three dropdowns don't quite have the same styling (see the carrot in the first dropdown).  This might be due to the datalist element

<img width="715" alt="Screen Shot 2021-11-17 at 5 11 57 PM" src="https://user-images.githubusercontent.com/35042815/142291370-1d803212-65be-483e-8ca5-2d58cec5659e.png">

Reply:
yes they do not have the same styling because of the reason you mentioned, I had previously played around with making these react-select components but the PO team preferred keeping these as these types of inputs

Reply:
can we make the first a datalist then so they're all the same?

Reply:
unfortunately we can't because cohort_type needs to be a select because it's an enum whereas the other 2 behave kind of similarly to assigned_users where there are suggestions but users can create their own values as well

Reply:
that's unfortunate... I wonder if you could hide the down carrot on the first one then so the styling before you open the dropdown at least looks uniform

Reply:
hmm I don't really think that's necessary because if we were to do that, we should do it for all our inputs in enrollment for consistency, which can be done in a future ticket

Reply:
actually, it looks like you can achieve this functionality with react select: https://react-select.com/creatable

Reply:
that's actually exactly what I used before but the PO team preferred the current implementation

Reply:
punting on this for now